### DISCUSSION TITLE
[Bug]: Azure OpenAI: DeploymentNotFound

### DISCUSSION THREAD

**Discussion by gaborvadasz** [danny-avila/LibreChat#1284] [Troubleshooting]
Hello!

I tried to configure Azure OpenAI according to the description, but no matter how I try I always get this:

Something is broken. Here is the specific error message we encountered: Failed to send message. HTTP 404 - {"error":{"code": "DeploymentNotFound", "message": "The API deployment for this resource does not exist. If you created the deployment in the last 5 minutes, please wait a moment and try again"}}

What can I do?

Thanks in advance!

Regards,
Gabor

**Comment by danny-avila** [comment]
Hi @gaborvadasz thanks for checking out the project!

Which description are you referring to? Please review the .env configuration guide: https://docs.librechat.ai/install/dotenv.html

This is a more condensed guide and may not have all the configuration details: https://docs.librechat.ai/install/apis_and_tokens.html

**Reply by gaborvadasz** [reply]
Of course, I went through the settings, the OpenAI version works, but not Azure OpenAI:

#============#
# Azure #
#============#

AZURE_API_KEY=*******
AZURE_OPENAI_MODELS=gpt-4
AZURE_OPENAI_DEFAULT_MODEL=gpt-4
# PLUGINS_USE_AZURE=true
AZURE_USE_MODEL_AS_DEPLOYMENT_NAME=TRUE
AZURE_OPENAI_API_INSTANCE_NAME=*******
AZURE_OPENAI_API_DEPLOYMENT_NAME=*******
AZURE_OPENAI_API_VERSION=2023-05-15
# AZURE_OPENAI_API_COMPLETIONS_DEPLOYMENT_NAME=
# AZURE_OPENAI_API_EMBEDDINGS_DEPLOYMENT_NAME=

**Reply by gaborvadasz** [reply]
is the region value missing somewhere?

**Reply by danny-avila** [reply]
try setting this to FALSE `AZURE_USE_MODEL_AS_DEPLOYMENT_NAME=TRUE` if the deployment name does not correspond with the modelname; e.g. 'gpt-35-turbo' for 'gpt-3.5-turbo'

**Reply by gaborvadasz** [reply]
It worked, my fault.

Unfortunately the docker restart librechat did not reload the .env file.
